FEATURES.md

```diff
@@ -26,15 +26,30 @@ noun rate|The rate of nouns across sentences |***noun_rate***|Stanza(66)|Yes|No|
 verb rate|The rate of verbs across sentences|***verb_rate***|Stanza(66)|Yes|No|String or JSON | - | -
 demonstrative rate|The rate of demonstrative across sentences|***demonstrative_rate***|Stanza(66)|Yes|No|String or JSON | - | -
 adjective rate|The rate of adjectives across sentences|***adjective_rate***|Stanza(66)|Yes|No|String or JSON | - | -
-pronoun rate|The rate of pronouns across sentences|***pronoun_rate***|Stanza(66)|Yes|No|String or JSON | - | -
```

Comment:
@Sarenne , did we remove pronoun rate from the table?

Reply:
It's just been moved but this list has now been updated again 👍